tests/config/nf-test.config

```diff
@@ -32,12 +32,6 @@ profiles {
         docker.enabled = true
         docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64'
     }
-    gpu {
-        docker.runOptions       = '-u $(id -u):$(id -g) --gpus device=0'
-        apptainer.runOptions    = '--no-mount tmp --writable-tmpfs --nv'
-        singularity.runOptions  = '--no-mount tmp --writable-tmpfs --nv'
-        use_gpu                 = true
-    }
     docker_self_hosted {
```

Comment:
the other gpu profile is outdated, needs platform argument

Reply:
So cut the other one?

Reply:
There's two on master though 😆 

https://github.com/nf-core/modules/blob/af84433c2e619cf28e026d4140b1b8763ae3690b/tests/config/nf-test.config#L35-L51